docs/source/examples/managed-jobs.rst

```diff
@@ -159,16 +115,89 @@ We can launch it with the following:
 SkyPilot will launch and start monitoring the job. When a spot preemption or any machine failure happens, SkyPilot will automatically
```

Comment:
L112 should refer to the new https://docs.skypilot.co/en/managed-jobs-docs-revamp/examples/managed-jobs.html#intermediate-storage-for-files support briefly?